pkgs/development/python-modules/mmcv/default.nix

```diff
@@ -0,0 +1,92 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytest
+, pythonOlder
+, cudaSupport ? false
+, cudatoolkit
+, torchCudaArchList ? "Kepler"
```

Comment:
I don't think we should have a single package depending on cuda arch settings like this.  We already have e.g. config.cudaCapabilities for configuring what architectures cuda-enabled packages are built for.

Reply:
if there is an example default.nix that will use it that you can point me to?

Reply:
I will try to use torch/default.nix as example

Reply:
done